website/pages/docs/getting-started.mdx

```diff
@@ -5,6 +5,9 @@ description: How to install and set up Chakra UI in your local project
 
 **First, you need to install Chakra UI**
 
+> If you're a `create-react-app` user, see the
```

Comment:
> If you're using `create-react-app`, see the

Maybe just me, but `a ... user` feels odd

Reply:
@ljosberinn How about "For `create-react-app` installation instructions," instead?

Reply:
Fine by me, feels a bit formal, but I don't mind.

Reply:
I agree but not sure how to state it better while keeping it short. I'll change it to that and if we come up with something better later, we can change it